robotframework_reportportal/service.py

```diff
@@ -34,10 +34,15 @@
 def to_epoch(date):
     """Convert Robot Framework timestamp to UTC timestamp."""
     try:
-        iso_date = isoparse(date)
+        iso_date = parse(date)
```

Comment:
Please, rename variable to something like `parsed_date = parse(date)`. We are not using `isoparse ` anymore.

Reply:
Done